application/controllers/digital_take_up.py

```diff
@@ -84,7 +131,27 @@ def channel_options(admin_client, uuid):
         if True in form.data.values():
```

Comment:
form.data.values will always contain elements, even when no channel choices (checkboxes) are selected. In the latter case, form.data.values will hold [False, False, False, False, False, False]. So bool(<list>) will still return True (that is, list contains elements) which is not want we want. Rather we want to ensure that at least one of the list elements is True (a checkbox has been ticked).